apps/ui/src/components/pages/manage/DashboardFilter.tsx

```diff
@@ -78,7 +78,7 @@ const DashboardFilter = ({ onFilterChange, filters, availableStates }: Dashboard
 		 *
 		 * Also if the user unselects all filters, default back to total.
 		 **/
-		if (selectedFilter === 'TOTAL' || (nextSelectedFilters.length === 0 && filters.length === 0)) {
+		if (selectedFilter === 'TOTAL' || (nextSelectedFilters.length === 0 && filters.length === 1)) {
```

Comment:
Is this correct? We only do this default when `filters.length === 1`?  I don't know the meaning of these checks, it just seems like an unusually specific case. I would've expected `filters.length > 0` or `filters.length === 0`, but specifically only doing this with one filter is unclear. Maybe explain in comment?